static/swagger/eds-spec.json

```diff
@@ -0,0 +1,392 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "PhotonIQ Global Event Delivery Service (EDS)",
+    "description": "The PhotonIQ Global Event Delivery Service (EDS) offers a robust platform capable of streaming live event data directly to your applications. This system supports thousands of simultaneous connections, allowing for easily incorporating event-driven capabilities into your infrastructure. This ensures your applications receive updates in real-time, enhancing their reactivity and efficiency.\n\nFeatures:\n- Delivery of live event information through WebSockets, including a unique feature for event de-duplication.\n- Options to subscribe to specific events and publish new ones to the event stream\n- Comprehensive metrics on user connections.\n- A health check and versioning API for the service.\nFor detailed information on endpoints, how to format requests and responses, and guidelines for integrating EDS with your applications, please refer to the API documentation provided below.\n",
```

Comment:
See comment from above.

Apply edits from the other file to this one. Why are there two? @joubert-macrometa, are there supposed to be two files? Should one be removed?

Reply:
Removed the swagger one.